packages/eslint-plugin/docs/rules/padding-line-between-statements.md

```diff
@@ -0,0 +1,32 @@
+# require or disallow padding lines between statements (`padding-line-between-statements`)
+
+## Rule Details
+
+This rule extends the base [`eslint/padding-line-between-statements`](https://eslint.org/docs/rules/padding-line-between-statements) rule.
+
+It adds support for TypeScript constructs such as `interface` and `type`.
+
+## How to use
+
+```jsonc
+{
+  // note you must disable the base rule as it can report incorrect errors
+  "padding-line-between-statements": "off",
+  "@typescript-eslint/padding-line-between-statements": [
+    "error",
+    {
+      "blankLine": "always",
+      "prev": "var",
+      "next": "return"
+    }
+  ]
+}
+```
+
+## Options
+
+See [`eslint/padding-line-between-statements` options](https://eslint.org/docs/rules/padding-line-between-statements#options).
```

Comment:
Let's call this out explicitly just so people can see it easily.

```suggestion
See [`eslint/padding-line-between-statements` options](https://eslint.org/docs/rules/padding-line-between-statements#options).

This extension rule adds the following additional `STATEMENT_TYPE` options:
- `interface`
- `type`
```

Reply:
The second line highlights these options. I have made it bold and added an example too.